rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -1099,7 +1117,15 @@ stream {
             {{ $externalAuth.AuthSnippet }}
             {{ end }}
 
+            {{ if and (eq $applyAuthUpstream true) (eq $applyGlobalAuth false) }}
```

Comment:
Need a comment as well here. What are we doing, and why are we doing? :)

As an example, why in this case we should force proxy_http_version = 1.1?

Reply:
Done